content/applications/hr/recruitment/add-new-applicants.rst

```diff
@@ -0,0 +1,169 @@
+- :guilabel:`Medium`: using the drop-down menu, specify how the job listing was found. The
+  pre-configured options are: :guilabel:`Banner`, :guilabel:`Direct`, :guilabel:`Email`,
+  :guilabel:`Facebook`, :guilabel:`Google Adwords`, :guilabel:`LinkedIn`, :guilabel:`Phone`,
+  :guilabel:`Television`, :guilabel:`Twitter` (now known as "X"), or :guilabel:`Website`. To add a
+  new :guilabel:`Medium`, type in the medium, then click :guilabel:`Create "(new medium)"`.
+- :guilabel:`Referred By User`: if referral points are to be earned for this job position in the
+  *Referrals* application, select the user who referred the applicant from the drop-down menu. The
+  *Referrals* application must be installed for this field to appear.
+- :guilabel:`Availability`: select the available start date for the applicant. To select a date,
+  click on the field to reveal a calendar. Use the :icon:`fa-angle-left` :guilabel:`(left)` and
+  :icon:`fa-angle-right` :guilabel:`(right)` arrows on either side of the month to navigate to the
+  desired month, then click the date. Leaving this field blank indicates the applicant can start
+  immediately.
+- :guilabel:`Tags`: select as many tags as desired from the drop-down menu. To add a tag that does
+  not exist, type in the tag name, then click :guilabel:`Create "new tag"`.
+
+Job section
+-----------
+
+The following fields are pre-populated when creating a new applicant, as long as these field are
+specified on the Job Position. Editing the fields is possible, if desired.
+
+- :guilabel:`Applied Job`: select the job position the applicant is applying to from the drop-down
+  menu.
+- :guilabel:`Department`: select the department the job position falls under from the drop-down
+  menu.
+- :guilabel:`Company`: select the company the job position is for using the drop-down menu. This
+  field only appears when in a multi-company database.
+
+Contract section
+----------------
+
+- :guilabel:`Expected Salary`: enter the amount the applicant is requesting for the role in this
+  field. The number should be in a `XX,XXX.XX` format. The currency is determined by the
+  localization setting for the company.
+- :guilabel:`Extra advantages...`: if any extra advantages are requested by the applicant, enter it
+  in the :guilabel:`Extra advantages...` field to the right of the :guilabel:`Expected Salary`
+  field. This should be short and descriptive, such as `1 week extra vacation` or `dental plan`.
+- :guilabel:`Proposed Salary`: enter the amount to be offered to the applicant for the role in this
+  field. The number should be in a `XX,XXX.XX` format.
+- :guilabel:`Extra advantages...`: if any extra advantages are offered to the applicant, enter it in
+  the :guilabel:`Extra advantages...` field to the right of the :guilabel:`Proposed Salary` field.
+  This should be short and descriptive, such as `unlimited sick time` or `retirement plan`.
+
+Application summary tab
```

Comment:
```suggestion
Application Summary tab
```
SUGGESTION: we tend to capitalize proper tab names even in section headings for easier readability